specification/cognitiveservices/data-plane/EntitySearch/v1.0/EntitySearch.json

```diff
@@ -0,0 +1,935 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "Bing Entity Search API",
+    "description": "The Entity Search API lets you send a search query to Bing and get back search results that include entities and places. Place results include restaurants, hotel, or other local businesses. For places, the query can specify the name of the local business or it can ask for a list (for example, restaurants near me). Entity results include persons, places, or things. Place in this context is tourist attractions, states, countries, etc.",
+    "version": "1.0"
+  },
+  "host": "api.cognitive.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/bing/v7.0",
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "name": "Ocp-Apim-Subscription-Key",
+      "type": "apiKey",
+      "in": "header"
+    },
+    "apiKeyQuery": {
+      "name": "subscription-key",
+      "type": "apiKey",
+      "in": "query"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    },
+    {
+      "apiKeyQuery": []
+    }
+  ],
+  "paths": {
+    "/entities": {
+      "get": {
+        "summary": "The Entity Search API lets you send a search query to Bing and get back search results that include entities and places. Place results include restaurants, hotel, or other local businesses. For places, the query can specify the name of the local business or it can ask for a list (for example, restaurants near me). Entity results include persons, places, or things. Place in this context is tourist attractions, states, countries, etc.",
+        "operationId": "entitiesGet",
```

Comment:
Yeah the search APIs are pretty straight forward, do let me know if this should be changed

Reply:
For what it's worth we'll eventually want to offer GET and POST of the same endpoint with roughly the same params

Reply:
interesting - what's gonna be the difference between both methods? Trying to be smart for the future is probably our main driver here: Is this single operation (or soon two) gonna be shipped as a standalone SDK? Or is it gonna be combined with other Bing Swaggers into one client (... so customers can do more than one single thing after acquiring their SDK :D ). In the latter case, I'd 100% recommend a name that will distinguish your operation from the others, e.g. `Entities_Get` or `Entities_Search`. Actually, I'd even recommend these now, at least I'd rather call `client.Entities.Search` than `client.EntitiesGet` which it would be right now ;)

Reply:
so yeah, TL;DR: my recommendation is hereby `Entities_Search`, you may of course adjust both segments to your liking, you probably know your customer expectations 🙂

Reply:
Fair enough. The main diff between GET and POST is to provide client the flexibility to send params more securely (i.e. no PII in query) as well as to allow them to send huge blobs of content that would break url length limits (i.e. a suuuper long query string)